var/spack/repos/builtin/packages/libmspack/package.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+class Libmspack(AutotoolsPackage):
+    """libmspack is a portable library for some loosely related Microsoft
+    compression formats"""
+    maintainers = ['vmiheer']
+    homepage = "https://www.cabextract.org.uk/libmspack/"
+    url = "https://www.cabextract.org.uk/libmspack/libmspack-0.10.1alpha.tar.gz"
```

Comment:
Why not use the git repository at https://github.com/kyz/libmspack/?  

Note the sha256 is different for the same version on GitHub (`bd8537b04ebd2149819ce867e3eba50f862a8a0043fd2061360b4101eddaa382`).